From: Alexander Graf <agraf@suse.de>
Date: Sun, 07 Apr 2013 00:16:00 +0100
Subject: [PATCH] ARM: xen: Add missing define
Patch-Mainline: No, required because of local xen patches

Our local Xen patches add a define that ARM doesn't export. Add it there too, so we can enable the Xen option in our kernel config.

Signed-off-by: Alexander Graf <agraf@suse.de>

Index: linux-3.9-rc5-master/arch/arm/include/asm/xen/interface.h
===================================================================
--- linux-3.9-rc5-master.orig/arch/arm/include/asm/xen/interface.h
+++ linux-3.9-rc5-master/arch/arm/include/asm/xen/interface.h
@@ -43,6 +43,7 @@ typedef uint64_t xen_ulong_t;
 
 /* Maximum number of virtual CPUs in multi-processor guests. */
 #define MAX_VIRT_CPUS 1
+#define XEN_LEGACY_MAX_VCPUS MAX_VIRT_CPUS
 
 struct arch_vcpu_info { };
 struct arch_shared_info { };
